src/helpers/ClauseResultsHelpers.ts

```diff
@@ -102,7 +102,14 @@ export function findAllLocalIdsInStatement(
         aliasMap[v] = statement.expression.localId;
         // Determine the localId for this alias.
         if (statement.localId) {
-          alId = statement.localId;
+          // Older translator versions require with statements to use the statement.expression.localId + 1 as the alias Id
+          // even if the statement already has a localId. There is not a clear mapping for alias with statements in the new
+          // translator, so they will go un highlighted but this will not affect coverage calculation
+          if (statement.type === 'With') {
```

Comment:
```suggestion
          if (statement.type === 'With' || statement.type === 'Without') {
```

Reply:
With testing, I was able to confirm that this behavior for `with` is the same for `without`, so we also need that check here in order for our highlighting to be backwards compatible.